plugins/keyboard-navigation/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "blockly-plugin-keyboard-navigation",
+  "name": "@blockly/keyboard-navigation",
   "version": "0.1.1",
-  "description": "A Blockly plugin.",
+  "description": "A Blockly plugin that adds keyboard navigation support.",
   "scripts": {
     "audit:fix": "blockly-scripts auditFix",
```

Comment:
You're going to need to add the predeploy script here as well to get the demo working. "predeploy": "blockly-scripts predeploy",